src/4.3_to_5.0/__snapshots__/migrate_43_to_50.test.ts.snap

```diff
@@ -2397,6 +2404,86 @@ exports[`migrate_43_to_50 returns a valid ActiveUI5 /ui folder on a small input
       },
     },
     "users": {
+      "children": {
```

Comment:
This change results from re-adding the `settings` to the `emptyLegacyUIFolder`. With the changes in this commit, this file remains unchanged in this PR.